packages/fxa-auth-server/test/remote/recovery_email_emails.js

```diff
@@ -851,14 +851,28 @@ describe('remote emails', function() {
         });
     });
 
-    it('fails to login', () => {
+    it('successful login', () => {
       return Client.login(config.publicUrl, secondEmail, password, {})
         .then(() => {
           assert.fail(new Error('should not have been able to login'));
         })
         .catch(err => {
+          // When logging in with a secondary email the auth-server returns the correct email
+          // to generate the authPW (based on OnePW protocol). This is the email that should be
+          // used to login.
```

Comment:
IIRC, one reason we haven't done this in the past is that it's a small unauthenticated info leak - if I know your secondary email address, I could do a fake login attempt and have the FxA server tell me your primary address. Have we / do we need to clear this info leak with security/privacy review?